pkg/monitoring/metrics/rbd_metrics_test.go

```diff
@@ -0,0 +1,94 @@
+package metrics
+
+import (
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	ioprometheusclient "github.com/prometheus/client_model/go"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubevirtv1 "kubevirt.io/api/core/v1"
+)
+
+const rbdDriver = "openshift-storage.rbd.csi.ceph.com"
+
+var _ = Describe("rbd_metrics", func() {
+	vm := &kubevirtv1.VirtualMachine{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-vm",
+			Namespace: "test-namespace",
+		},
+	}
```

Comment:
I think this is not a good design, because changes to the `vm` variable in one test case will be visible in another test case.
Can you instead use `BeforeEach()` to create a new `vm` instance for every test?

Reply:
updated